discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -48,7 +48,7 @@ def _terraform_params(self, **kwargs):
                   "libvirt_master_memory": utils.resource_param(kwargs.get("master_memory", 16984), "master_memory", operators),
                   "libvirt_worker_vcpu": utils.resource_param(kwargs.get("worker_vcpu", 4), "worker_vcpu", operators),
                   "libvirt_master_vcpu": utils.resource_param(kwargs.get("master_vcpu", 4), "master_vcpu", operators),
-                  "worker_count": kwargs.get('num_workers', 0),
+                  "worker_count": utils.ocs_resource_param(kwargs.get('num_workers', 0), "num_workers", operators),
```

Comment:
I don't think it should specifically mention `ocs`. Why not use the same `resource_param`?

Reply:
The logic is a bit different we could add `OCS` specific requirements to other operator with 0 value. Do you suggest to do this instead?

Reply:
What is the specific requirement? additional disks? If so, then yes.
I don't want to have "OCS" reference in the terraform generic conf.

Reply:
Additional disks and number of workers. We need to make sure we have at least 3 when deploying `OCS`

Reply:
Okay. It's fine to add zeros to others